src/Microsoft.Data.Sqlite/Microsoft.Data.Sqlite.csproj

```diff
@@ -1,34 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\build\common.props" />
   <PropertyGroup>
-    <Description>SQLite implementation of the System.Data.Common provider model.
-
-Commonly Used Types:
-Microsoft.Data.Sqlite.SqliteCommand
-Microsoft.Data.Sqlite.SqliteConnection
-Microsoft.Data.Sqlite.SqliteConnectionStringBuilder
-Microsoft.Data.Sqlite.SqliteDataReader
-Microsoft.Data.Sqlite.SqliteException
-Microsoft.Data.Sqlite.SqliteFactory
-Microsoft.Data.Sqlite.SqliteParameter
-Microsoft.Data.Sqlite.SqliteTransaction</Description>
-    <TargetFrameworks>net451;netstandard1.3</TargetFrameworks>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <PackageTags>SQLite;Data;ADO.NET</PackageTags>
-    <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)$(MSBuildProjectName).ruleset</CodeAnalysisRuleSet>
-    <IncludeSymbols>true</IncludeSymbols>
+    <Description>Metapackage that includes Microsoft.Data.Sqlite.Core and SQLitePCLRaw.bundle_green.</Description>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <TargetFrameworks>net451;netstandard1.2</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
-    <None Include="stylecop.json" />
-    <None Include="$(CodeAnalysisRuleSet)" />
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="SQLite" Version="3.12.3" />
-    <PackageReference Include="StyleCop.Analyzers" Version="1.0.0">
+    <PackageReference Include="SQLitePCLRaw.bundle_green" Version="1.1.2" />
```

Comment:
Set `PrivateAssets=None`. SQLitePCLRaw.bundle_green has some build targets that should be included. Cref https://github.com/NuGet/Home/issues/4521